CODEOWNERS

```diff
@@ -185,7 +185,7 @@
 /gallery/apimanagement-service/** @microsoft/azure-api-management
 /gallery/workbook/microsoft.mobilenetwork-mobilenetwork-sites.json @microsoft/azure-private-mec-workbooks
 /gallery/workbook/microsoft.apimanagement-service.json @microsoft/azure-api-management
-/gallery/workbook/microsoft.experimentation-experimentationworkspaces @microsoft/exp-devs
+/gallery/workbook/Microsoft.Experimentation-experimentWorkspaces @microsoft/exp-devs
```

Comment:
the other gallery files+folders are all lower case.  hypothetically our build/packaging just `tolower`'s everything to be consistent downstream but you may also want to do that to be consistent?  (i want to say client side it also does this?)

Reply:
implemented recommendation